cli.js

```diff
@@ -3,13 +3,35 @@
 import meow from 'meow';
 import githubMarkdownCss from './index.js';
 
-meow(`
+const cli = meow(`
 	Usage
 	  github-markdown-css > <filename>
+
+	Options
+	  --light     Light theme
+	  --dark      Dark theme
+	  --list, -l  List available themes
```

Comment:
I don't see the point of a short flag for this. It's not a command people will run often.